browser/src/control/jsdialog/Widget.Timefield.js

```diff
@@ -0,0 +1,54 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * JSDialog.Timefield - icon view widget
+ *
+ * Example JSON:
+ * {
+ *     id: 'time',
+ *     type: 'time',
+ *     text: '01:01:01'
+ * }
+ */
+
+/* global JSDialog $ */
+
+JSDialog.timeField = function (parentContainer, data, builder) {
+	var container = L.DomUtil.create('span', builder.options.cssClass + ' ui-timefield-container', parentContainer);
+	if (data.id !== undefined) {
+		container.setAttribute('id', data.id);
+	}
+
+	var inputTimeField = L.DomUtil.create('input', builder.options.cssClass + ' ui-timefield', container);
+	inputTimeField.setAttribute('type', 'time');
+	inputTimeField.setAttribute('step', 1); // forces the display of seconds
+	inputTimeField.id = data.id;
+	inputTimeField.value = data.text;
+
+	inputTimeField.addEventListener('change', function (event) {
+		var timefield = event.target;
+
+		var attrdisabled = timefield.getAttribute('disabled');
+		if (attrdisabled !== 'disabled') {
+			var valueMSecs = timefield.valueAsNumber * 1000;
+			builder.callback('spinfield', 'change', timefield, timefield.value, builder);
+			console.log('changed', event, timefield.valueAsNumber, valueMSecs, timefield.value);
```

Comment:
builder.callback will log the message we send, is that console.log needed? At least please make it console.debug so it will not appear in logs when not needed.